### ISSUE TITLE
[BUG] VAC Audio Stream startup error

### ISSUE THREAD

**Issue by torrenthd** [sjk7/Thetis-2.9.0#57]
1) VAC Audio Stream startup very inconsistent... failing to startup from time to time in my main computer with a SoundBlaster sound card.

2) Unable to startup using Windows WDM-KS drivers (worked for a while) on a new computer that I'm setting up with an embeded Realtek AC1200 sound card:

![IMG_20230511_183037](https://github.com/sjk7/Thetis-2.9.0/assets/114567653/389db4ea-e821-4b41-a85c-02eb39372d9f)

**Comment by torrenthd** [comment]
Deep cleaned system from old Realtek drivers, installed again the new ones and now seems that Windows WDM-KS work fine...

**Comment by sjk7** [comment]
Hello and thanks,

Yup, this kind of thing is usually driver-related. I wish PortAudio gave better error messages, though. For a start it never tells you if the input or the output is broken ...

I like Kernel Streaming very much. It has lower latency for me than even ASIO.

Shall I close this?

Steve

**Comment by torrenthd** [comment]
Not really, today had 2 more issues in my main PC running SoundBlaster ASIO drivers.

Never had that kind of problems with Richie's versions, so looks like it is due to some changes introduced in your branch.
It is weird, I can have no problems for a few days and suddenly 2 issues the same evening.

It is very annoying because I must restant my computer to solve the issue.

**Comment by sjk7** [comment]
Well, remember we use the latest portaudio here, so it's very likely -- its
worth checking to see if there are any similar issues reported on the
portaudio git repo. I do know of some issues with certain APIs that I do
need to report to them, once I am sure the problem is in the portaudio dll
and not in my code,

Steve

**Comment by torrenthd** [comment]
For sure, I had to change my portaudio settings from 0 to 1 to get rid of this issue. I had errors  starting up VAC 75% of the time.
It was rock solid, but unfortunately,  just for a few days... right now, with portaudio set to 1, I'm still facing some startup errors.  

Could you please point me to the portaudio git repo?

**Comment by sjk7** [comment]
Hello,

Here you go:
PortAudio (github.com) <https://github.com/PortAudio>

Steve
